api-contracts/openapi/openapi.yaml

```diff
@@ -94,6 +94,8 @@ paths:
     $ref: "./paths/tenant/tenant.yaml#/getQueueMetrics"
   /api/v1/tenants/{tenant}/events:
     $ref: "./paths/event/event.yaml#/withTenant"
+  /api/v1/tenants/{tenant}/events/bulk-create:
```

Comment:
i think this should just be POST .../events/bulk

Reply:
also do we have a POST /events for non-bulk insert? im pretty sure we only have a grpc for this... would be good to add a rest endpoint

Reply:
I thought we did 
  /api/v1/tenants/{tenant}/events:
    $ref: "./paths/event/event.yaml#/withTenant"
    
and in event.yaml in "WithTenant" a post looks like it creates
      post:
    x-resources: ["tenant"]
    description: Creates a new event.
    operationId: event:create
    parameters:
      - description: The tenant id
        in: path
        name: tenant
        required: true
        schema:
          type: string
          format: uuid
          minLength: 36
          maxLength: 36
    requestBody:
      content:
        application/json: